source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,281 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
+#############################################
+
+Upstream Clear Linux offers a plethora of bundles to choose from.  
+As of version 32570, there are over 1000 bundles and counting.  
+
+To provide you access to even more software outside of the distro, we have 
```

Comment:
The first person voice is not in line with the rest of the (technical) documentation (i.e. avoid using "we").

Reply:
I used to avoid "we" until I was told to.  "We" should bring it up at the Docs. meeting to get consensus.

Reply:
All "we"s eliminated!